bears/rust/RustClippyLintBear.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.CargoRequirement import (
+        CargoRequirement)
+
+
+class RustClippyLintBear(GlobalBear):
+    LANGUAGES = {'Rust'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {
+        'Formatting',
+        'Unused Code',
+        'Syntax',
+        'Unreachable Code',
+        'Smell',
+        'Code Simplification',
+    }
+    EXECUTABLE = 'cargo.exe' if os.name == 'nt' else 'cargo'
+    REQUIREMENTS = {
+        CargoRequirement('clippy')
+    }
+    SEVERITY_MAP = {
+        'warning': RESULT_SEVERITY.NORMAL,
+        'error': RESULT_SEVERITY.MAJOR,
+    }
+
+    def run(self):
+        args = ('clippy', '--quiet', '--color', 'never',
+                '--', '-Z', 'unstable-options',
+                '--error-format', 'json',
+                '--test')
```

Comment:
Done. It doesn't do JSON out of the box, but `output_format=None,` lets one customize the behaviour. I'll make an issue about this.